src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -198,10 +210,10 @@ const PermissionIdentities: FC = () => {
     defaultSort: "name",
   });
 
-  // NOTE: tls user group membership cannot be modified, this will be supported in the future
-  const nonTlsUsers = identities.filter((identity) => {
-    const isTlsIdentity = identity.authentication_method === "tls";
-    return !isTlsIdentity;
+  const nonLegacyIdentities = identities.filter((identity) => {
```

Comment:
can we call this `fineGrainedIdentities` to be consistent?

Reply:
restricted can be a bit confusing, since legacy certificates can still be restricted by projects.